iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/BusAdderImpl.java

```diff
@@ -33,7 +35,8 @@ protected String getTypeDescription() {
     @Override
     public ConfiguredBus add() {
         String id = checkAndGetUniqueId();
-        ConfiguredBusImpl bus = new ConfiguredBusImpl(id, voltageLevel);
+        String name = Optional.ofNullable(getName()).orElse(id);
```

Comment:
Do you add this for non regression purpose? This is related to the issue about handling optional names

Reply:
This is redundant with what is done in Identifiable.getId, this line could be removed, no?

Reply:
Fixed, I didn't check if `name` was optional for `Identifiable`